app/src/androidTest/java/net/skyscanner/backpack/button/BpkButtonLinkTest.kt

```diff
@@ -0,0 +1,104 @@
+package net.skyscanner.backpack.button
+
+import android.widget.FrameLayout
+import androidx.core.content.ContextCompat
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import net.skyscanner.backpack.BpkSnapshotTest
+import net.skyscanner.backpack.createThemedContext
+import net.skyscanner.backpack.demo.R
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+
+/**
+ * FIXME
+ * 31/10/2018
+ * see https://github.com/Skyscanner/backpack-android/pull/103
+ *
+ * The test cases with icons have been ignored. The snapshot test framework is not rendering
+ * the icons when using `setCompoundDrawablesRelativeWithIntrinsicBounds` instead of
+ * `setCompoundDrawablesWithIntrinsicBounds` on API-21 which is required to support RTL. We can add
+ * them when we update the emulators on the CI or move to a device farm for snapshot testing
+ */
+
+class BpkButtonLinkTest : BpkSnapshotTest() {
+
+  @Before
+  fun setup() {
+    setDimensions(300, 300)
```

Comment:
Can we create more specific dimensions to match the button?

Reply:
I'll reduce a bit but in general I prefer to keep a bigger frame so we have space to add more things if needed

Reply:
Ok